### ISSUE TITLE
RPM build fails when installing nginx-ldap-auth.logrotate

### ISSUE THREAD

**Issue by lesinigo** [nginxinc/nginx-ldap-auth#33]
Trying to build an RPM from the current master I get:
```
+ install -m644 nginx-ldap-auth.logrotate /root/rpmbuild/BUILDROOT/nginx-ldap-auth-0.0.3-1.el7.centos.x86_64/etc/logrotate.d/nginx-ldap-auth
install: cannot create regular file '/root/rpmbuild/BUILDROOT/nginx-ldap-auth-0.0.3-1.el7.centos.x86_64/etc/logrotate.d/nginx-ldap-auth': No such file or directory
error: Bad exit status from /var/tmp/rpm-tmp.dF42L5 (%install)
```

**Comment by vl-homutov** [comment]
Thanks for reporting, we'll fixe the issue with directories.

**Comment by lesinigo** [comment]
it is already fixed (at least for me) by pull request #34, please take a look at that and merge it if it looks good to you

**Comment by oxpa** [comment]
pull request is accepted.
Spec won't work for non-systemd distro, but that is a separate issue.
